README.md

```diff
@@ -610,7 +610,7 @@ products
 
 ```js
 
-// aka: the father of map and filter
+// aka: the parent of map and filter
```

Comment:
This is probably the only opinionated change, I can remove it if you want, I'll totally understand 😊